apps/cdk/src/constants.ts

```diff
@@ -0,0 +1 @@
+export const zoneName = 'antalmanac.com';
```

Comment:
Nit but I think this can just be hardcoded in the two places its used because it will literally never ever change so dunno if it is worth extracting to a variable

Reply:
Oh so my motivation for this is literally, so the stacks look more similar lol. The `frontend` and `backend` stacks are structured very similarly, but the `zoneName` placement would be different in them, since it needs to be declared at different places. So I just put it in a separate file lol :nerd_face:

Reply:
if anything I feel like you could've just extracted the entire zone component (not that that is really worth it either) but obviously not a big deal either way.